tests/acceptance/support/sandbox.rs

```diff
@@ -462,8 +462,8 @@ fn volta_tools_dir() -> PathBuf {
 fn inventory_dir() -> PathBuf {
     volta_tools_dir().join("inventory")
 }
-fn user_dir() -> PathBuf {
-    volta_tools_dir().join("user")
+fn default_dir() -> PathBuf {
+    volta_tools_dir().join("default")
```

Comment:
We definitely want to keep the argument to `join()` as `"user"`, since that represents the path on the filesystem. I'm split on whether we want to update the name of this function, however. I don't think we want to keep it as `default_dir`, since that implies it's the default of something, but we could either leave it as `user_dir` or maybe do something like `default_tool_dir`

Reply:
in this case I'd like to keep as `user_dir`